src/Symfony/Bundle/FrameworkBundle/Command/AbstractConfigCommand.php

```diff
@@ -48,7 +50,8 @@ protected function findExtension($name)
             // input is bundle name
 
             if (isset($bundles[$name])) {
-                $extension = $bundles[$name]->getContainerExtension();
+                $bundle = $bundles[$name];
```

Comment:
`$bundle` is not initialized before, but you rely on it in the outer scope. I'd suggest to initialize it to `null`, just like we do with the [`$extension`](https://github.com/aitboudad/symfony/blob/config_cmd/src/Symfony/Bundle/FrameworkBundle/Command/AbstractConfigCommand.php#L45).